src/docs/resources/faq.md

```diff
@@ -206,11 +206,11 @@ of Flutter, read the [architectural overview].
 
 ### How does Flutter run my code on Android? {#run-android}
 
-The engine's C and C++ code are compiled with Android's NDK.
+The engine's C and C++ codes are compiled with Android's NDK.
```

Comment:
Pluralizing "codes" seems to be an English dialectical thing. We don't generally do that.